bot/interact.go

```diff
@@ -81,6 +81,10 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 					if minSalary > maxSalary {
 						validationErrors["min_salary"] = "The Salary Min field should contain a lower value than the specified in Salary Max field."
 					}
+
+					if (minSalary * 2) > maxSalary {
+						validationErrors["max_salary"] = "The gap between MinSalary and MaxSalary is rather large. Maybe you should post two different job offers with different responsibilities and required qualifications. Salary is a relevant field, we recommend you try to keep it meaningful to increase the chances of taking the position seriously by potential candidates."
```

Comment:
```suggestion
					if (minSalary * 2.5) > maxSalary {
						validationErrors["max_salary"] = "The gap between MinSalary and MaxSalary is rather large. Maybe you should post two different job offers with different responsibilities and required qualifications. Salary is a relevant field, we recommend you try to keep it meaningful to increase the chances of taking the position seriously by potential candidates."
```

Reply:
cc @linkita @smoya @KoeSystems (rest agrees)

Reply:
@gonzaloserrano my GoLang _fu_ is non-existant and this change had no test so my first draft to move from `2x` to `2.5x` was using:  `(minSalary * 5) > (maxSalary * 2)`.

I trust you know what you're doing. :-D

Reply:
> I trust you know what you're doing. :-D

I have no idea, just changed the value XD

Reply:
well, and also the type... I'll double check

Reply:
it does now work 

> interact.go|81 col 12| constant 2.5 truncated to integer 🤦

Reply:
LOL
YOLO based programming, test in prod, hahaha

I'm not at my keyboard ATM. I'll Push my draft later.